src/source/campaign-linkedads.rst

```diff
@@ -0,0 +1,31 @@
+.. _campaign_linkedads_overview:
+
+Campaign Linked Ads
+===================
+
+.. _campaign-linked-ads:
+
+
+Campaign's linkedAds component defines which ads are part of the campaign. It is possible to add / remove (link / unlink) ads from a campaign using
+:ref:`put_campaign_linkedads` / :ref:`delete_campaign_linkedads`. Deleting a linkedAd from a campaign does not remove the ad from your inventory, it merely takes it offline in the context of the particular campaign.
+
+LinkedAds are stored as part of the campaign, and have a reference to an Ad using its ID. A linkedAd consists of two parts:
+the Ad ID and some possible overrides from the campaign defaults.
+Using linkedAds, it is possible to add a single ad to multiple campaigns (which may have different budgets, CPC and targeting criteria).
```

Comment:
talking about multiple campaigns here; do we want to expose that info already? If we do, we should at least mention that this is still uncertain when and if that gets released.

Reply:
this part is not surfaced to the docs yet :)